### ISSUE TITLE
sitemap problems

### ISSUE THREAD

**Issue by alvin-wu03** [KarlWithK/EandF#2]
We like how your sitemap is detailed, but it's a little too detailed. It's a bit hard to follow because of all the different arrows going in every direction. A more linear design might have been better.

**Comment by SayaDuck** [comment]
Adding on, it's probably possible to just rearrange the boxes is a more intuitive fashion without having to modify any contents.

**Comment by amelia-chin** [comment]
thanks, but we believe that our sitemap works for us, so we didn't change it
